Sources/TuistGraph/Models/PluginStencil.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import TSCBasic
+
+/// Stencil plugin model
+public struct PluginStencil: Equatable {
```

Comment:
Why does this need to be in `TuistGraph` module?

Reply:
I looked again today and it seems to be a module that has nothing to do with `TuistGraph` and has something to do with `TuistLoader`. 

Can i move that module to `TuistLoader/Models`?